pysmiles/smiles_helper.py

```diff
@@ -38,14 +38,12 @@
                           STEREO_PATTERN + HCOUNT_PATTERN + CHARGE_PATTERN +
                           CLASS_PATTERN + r'\]$')
 
-AROMATIC_ATOMS = "B C N O P S Se As *".split()
+ELECTRON_CONFIG_PATTERN = (r'(?:\[[A-Z][a-z]?\])?((?:'
+                           r'(?:[1-9]s(?P<s>[1-2]))|(?:[1-9]p(?P<p>[1-6]))|(?:[1-9]d(?P<d>10|[1-9]))|(?:[1-9]f(?P<f>1[0-4]|[1-9]))'
+                           r'){1,4})(?:\(predicted\))?')
```

Comment:
in 2 weeks no one can remember what this does or is. I'd suggest adding some minimal comments on what it does. I suppose it converts something like 2s3p to the valency??